appsflyer/build.gradle

```diff
@@ -22,10 +22,10 @@ android {
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
     // rudder core sdk
-    compileOnly 'com.rudderstack.android.sdk:core:1.+'
+    implementation 'com.rudderstack.android.sdk:core:1.2.1'
```

Comment:
Don't use specific version, use minumum version like 1.2.1+

Reply:
Implemented the changes.

Reply:
Again specified a particular version.!

Reply:
I've specified like this: **[1.2.1,)**. Please refer here: https://stackoverflow.com/a/34879252/15947241

Reply:
It means all version starting from 1.2.1

Reply:
Oh, okay.